game/src/info/trip.rs

```diff
@@ -332,6 +355,33 @@ pub fn finished(
             Line(trip.purpose.to_string()).secondary().into_widget(ctx),
         ]));
     }
+    // TODO Duplicating some code from ongoing() until we decide how to consolidate things.
```

Comment:
All of the code in this file is kind of a mess; when we have some of the design figured out, I'm going to make a refactoring attempt